abis/EthConversionProxy.json

```diff
@@ -175,5 +240,31 @@
     "outputs": [],
     "stateMutability": "payable",
     "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "_chainlinkConversionPathAddress",
+        "type": "address"
+      }
+    ],
+    "name": "updateConversionPathAddress",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "_paymentProxyAddress",
+        "type": "address"
+      }
+    ],
+    "name": "updateConversionProxyAddress",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
   }
```

Comment:
@yomarion when using the script, the EthConversionProxy abi v1 has been updated and is now the same as the v2. Is that the expected behaviour ?